components/brave_wallet_ui/stories/mock-data/mock-wallet-state.ts

```diff
@@ -171,6 +171,57 @@ const mockCurrencies: BraveWallet.OnRampCurrency[] = [
   }
 ]
 
+const mockCoinMarketData: BraveWallet.CoinMarket[] = [
+  {
+    id: 'bitcoin',
+    symbol: 'btc',
+    name: 'Bitcoin',
+    image: 'https://assets.coingecko.com/coins/images/1/large/bitcoin.png?1547033579',
```

Comment:
Is this mock data working now that the CSP has been updated?

Reply:
@kdenhartog  This mock data is used in storybook which we use for developing frontend components(meant strictly for development not production). So this is okay.

Reply:
Can you point in the `CSP` code where we allow only for `assets.cgproxy.brave.com`?

Reply:
This is used in storybook. It will not work when proxied. Also, it does not need proxying because it is only used during development.